mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx

```diff
@@ -89,6 +82,12 @@ import {
   ButtonGroupTabKey,
   SwapButtonGroup,
 } from "./components/SwapButtonGroup";
+import { TokenDropdownButton } from "./components/TokenDropdownButton";
+import { ActiveUSDValueV2 } from "../../Loans/VaultDetail/components/ActiveUSDValueV2";
+import {
+  WantFutureSwapRow,
```

Comment:
Why is this Want? Let's rename it to From or To

Reply:
Updated! was torned if should name it based on the UI label or its functionality 
Updated to ```InstantSwapRowTo and FutureSwapRowTo```